src/auth/authGroups.ts

```diff
@@ -0,0 +1,21 @@
+import { AuthGroup, AuthGroupName } from './authGroup'
+import { RecordStepPermission } from './permission'
+
+const isSystemAdminGroup = (authGroup: AuthGroup): boolean => authGroup.name === AuthGroupName.systemAdmin
+
+const getRecordSteps = (authGroup?: AuthGroup): { [key: string]: RecordStepPermission } | undefined =>
+  authGroup?.recordSteps
+
+const getRecordEditLevel = (step?: RecordStepPermission) => (
```

Comment:
step is the step number (string?)
it shouldn't be optional

Reply:
why is step number a string? should it be a number?